### PR TITLE
Shotgun nerf / rebalance

### PR DESCRIPTION
**Pull request by PowerfulBacon**
I know this isn't likely to be popular, but in their current state shotguns are like the debtor's punch ability and are in dire need of a rebalance.

## About The Pull Request

Rebalances the stats of shotguns to make them more similar to other ranged weapons in the game, although on almost all shells this is a nerf due to the power of a majority of shells.

**Calculations:**

All calculations assume a target's hitbox takes up a full tile (Which it does) and works by calculating the probability for each individual pellet hitting at a certain range, then multiplying it by the damage.

**Damage Calculation:**

Edit: Actually I just noticed the calculation was the version 1, it should say -xpt instead of -xt

image

**Spreadsheet:**

image
(Negative damages are due to being too lazy to clamp value (Projectiles are deleted once they go below 0 damage, so they would deal 0 rather than heal the player).)

**Changes:**
Buckshot: Reduced raw damage from 12.5 to 9. Decreased spread from 25 to 10 degrees.
Beanbag: No Changes
Soporific: First shot now applies 8 seconds of confusion. If hitting a confused target, will put them to sleep for 5 seconds (Old effect: 5 seconds sleep).
Incendiary: No changes
Dragon's Breath: No changes
Stun Slug: Unused, no changes
Meteor Slug: Hard stun reduced from 8 seconds to 2 seconds. Stun is applied if the target hits a wall due to knockback of the shell.
Frag12: 5 seconds of paralysis reduced to 1 second. Not that it really matters since these are generally going to be a 1 hit kill from the explosion or at least limbs will be lost.
Rubbershot: Reduced stamina damage from 11 to 9. Reduced variance from 25 to 20
Incapacitating Shell: Reduced stamina damage from 6 to 5. Reduced variance from 25 to 50
Improvised Shell: No changes
Laser Slug: Reduced damage from 15 to 10. (This thing was ridiculous before and actually pretty easy to make)
Slugs: Given -60 armour penetration making them less effective against armoured targets.

Shotguns: Pump-action shotguns require your second hand to be free to pump the pump (You can still fire with only 1 hand).

Main changes were to shells that hard-stun and pellet based shells with a high spread.

## Why It's Good For The Game

**Opinion:**

The 2 main types of combat in the game, ranged and melee have very different approaches and balance to make different things effective at these ranges. Shotguns follow the balance of the ranged approach to combat, however act more similar in the melee end of things, since they aren't effective at range and are only really useful in close quarters.

Shotguns in their current state are usually an instant win, even when caught off guard, which generally isn't that fun. This PR aims to solve this by changing shotguns to a close-range weapon, without knowing that you can win a fight just by bum-rushing into melee range and 1 tapping your opponent.

Graph of buckshot changes:
image
Red represents previous damage, blue represents new damage. (Damage is on the Y access, so while it may seem that the lines are not too far apart the new damage is significantly higher at mid ranges.)

The changes to shotgun pump and shell stats were made to make them a little riskier in close-quarter combat since an item that blocks all melee attacks and an item that can deal a significant amount of damage that alone could make melee combat at melee range insignificant was extremely poor to play against. The goal is to make shotguns high-risk high-reward, rather than low-risk high-reward and to encourage shotguns to stay close to a target but still a little out of melee range (2-3 tiles for best effects). Due to the calculations, damage at 1-2 range has been reduced while at 3-5 range has been increased which heavily encourage this playstyle; hopefully, this will promote the use of shotguns as a ranged weapon rather than playing shotguns with a melee weapon playstyle (Since in melee combat currently shotgun overshadow basically all melee weapons).

## Changelog
:cl:
balance: Buckshot - Reduced raw damage from 12.5 to 9. Decreased spread from 25 to 10 degrees.
balance: Soporific - First shot now applies 8 seconds of confusion. If hitting a confused target, will put them to sleep for 5 seconds (Old effect: 5 seconds sleep).
balance: Meteor Slug - Hard stun reduced from 8 seconds to 2 seconds. Stun is applied if the target hits a wall due to knockback of the shell.
balance: Frag12 - 5 seconds of paralysis reduced to 1 second. Not that it really matters since these are generally going to be a 1 hit kill from the explosion or at least limbs will be lost.
balance: Rubbershot - Reduced stamina damage from 11 to 9. Reduced variance from 25 to 20
balance: Incapacitating Shell - Reduced variance from 25 to 20, stamina damaged reduced from 6 to 5
balance: Laser Slug - Reduced damage from 15 to 10. (This thing was ridiculous before and actually pretty easy to make)
balance: Shotguns - Pump-action shotguns now require the pumping hand to be empty.
balance: Mech Scattershot - Pellet damage reduced from 24 to 18.
balance: Slugs now have an armour penetration of -60, making them very effective against unarmoured targets but not armoured targets.
/:cl:

### CODE REVIEW

**Comment by Sapfearon** [comment]
another month another shotgun nerf. Before you start nerfing station weapons nerf antags first. Shotguns are literally only ballistic tool in station arsenal, stop nerfing them without offering anything in return.

**Comment by Sinestia** [comment]
> another month another shotgun nerf. Before you start nerfing station weapons nerf antags first. Shotguns are literally only ballistic tool in station arsenal, stop nerfing them without offering anything in return.

This comes across as more of an antag nerf to me honestly, it's mostly them who can get the most use out of shotguns at least on sage

**Comment by PowerfulBacon** [comment]
> another month another shotgun nerf. Before you start nerfing station weapons nerf antags first. Shotguns are literally only ballistic tool in station arsenal, stop nerfing them without offering anything in return.

Nanotrasen mainly use energy weapons, but as for guns you still have access to:
Autorifles, combat shotguns, riot shotguns, surplus rifles, energy guns, laser guns. This PR doesn't make shotguns completely useless and was made to make them more in line with the balance of combat at the range they are intended for. Shotguns arent useless now, they just arent overpowered.

**Comment by nikk735** [comment]
shotguns were shit they hold like 5 shots and need to be manually pumped meaning like 1 shot a second, imo rubbershot needed to be buffed and meteor and frag12 needed to be buffed they were absolutely useless actually used when compared to any other shot. I have not seen anyone put the effort into making these shells except for making pulse with the goal of tiding.

**Comment by PowerfulBacon** [comment]
> 
> 
> shotguns were shit they hold like 5 shots and need to be manually pumped meaning like 1 shot a second, imo rubbershot needed to be buffed and meteor and frag12 needed to be buffed they were absolutely useless actually used when compared to any other shot. I have not seen anyone put the effort into making these shells except for making pulse with the goal of tiding.

I made them, they are very easy to make and very overpowered. The reason they aren't used more is because a lot of people don't know about them, but the laser scatter does 90 damage, meteor shell hardsuits for 8 seconds and the frag12 while hard to make is essentially an instakill (this PR doesn't change that, because thats how explosions work not shotguns). The main one targetted was buckshot. Almost all changed shells are less effective at point blank range, however are more effective at range. (They are guns, not melee weapons after all, so shouldn't have the stats to be played only as melee style weapons)

**Comment by nikk735** [comment]
the laser was pretty good thats all ive seen used properly, the other two ive seen used but they didn't seem to do damage at all, people hit would just get stunned and run away

**Comment by BrotherHangyul** [comment]
Big bruh. Shotguns are sec's only cure for sudden nukies, blood cult and basically any antag with some sort of energy resistance. Outside of war ops, realistically it'll take 13 minutes to get some ballistics (autorifles, surplus rifles etc) from cargo which is more than enough time for the antag to kill a whole buncha people. Shotguns are meant to be powerful at close range, it's like getting mad at the sniper rifle for being good at long range.

**Comment by PowerfulBacon** [comment]
> Big bruh. Shotguns are sec's only cure for sudden nukies, blood cult and basically any antag with some sort of energy resistance. Outside of war ops, realistically it'll take 13 minutes to get some ballistics (autorifles, surplus rifles etc) from cargo which is more than enough time for the antag to kill a whole buncha people. Shotguns are meant to be powerful at close range, it's like getting mad at the sniper rifle for being good at long range.

Shotguns aren't the only cure for a sudden cult. They are the most effective (by far). There is a huge arsenal of unused weapons, but that isn't really considered often when you can grab a shotgun and go out to near 1 tapping every single antagonist on the station.
Shotguns are way too effective as a close-quarters weapon and make almost all melee weapons completely ineffective (Which is the most common antagonist weapon). Shotguns should still be powerful close range, however shouldn't be uncounterable/powerful to the point where the meta strategy is to run up to the people with melee weapons until you are in range at them because you know you will win no matter what.
The ability to have proper fights, rather than the 1 hit-win most security mains like is usually not as fun for security, but overall it leads to more engaging combat and better-balanced rounds, rather than the immediate stomping on any threat once the shotgun is pulled out. For larger threats like blood cult, revs and the round carrying modes, security should be going to cargo to arm up and get a good arsenal.

> shotguns aren't securities only cure for sudden nukies, blood cult and basically any antag

That is exactly where the problem lies: Shotguns outmatch every single antag and there is no point to using anything else. They are a cure for sudden nukies, blood cult etc., but to the robust security member they aren't the only cure and the weapons chosen by security should suit the situation rather than being shotgun for curing every single antag possible.

**Comment by BrotherHangyul** [comment]
Yeah? A shotgun should outmatch a melee weapon? By like definition?

They're **literally** designed as close-quarters firearms, they're meant to outmatch a melee weapon. Only a few antagonists rely on melee weapons anyway, that being Slaughter Demons (who can just crawl away from the shotgun), Cult (who has the mirror shield and therefore can only be hurt by slugs, which take good skill to aim with and deal nowhere near as much damage) and e-sword traitors, who could just be sneaky or by a revolver, a literal backpack shotgun (60 brute, same as shotgun slugs).

Ninjas have ninja-stars, lings can regenerate. Shotguns are meant to outmatch melee weapons, and even then antags who use melee weapons have quite a few counters to them with enough reaction time.

Shotguns aren't even a one-hit win they're two hits. 

Shotguns are also adaptable as fuck, with different rounds for different antags; slugs for nukies and culties, buckshot for revs / tots,  incendiary for lings/blob and sleepy for basically everything esle.

Stop trying to nerf shotgun for doing its job.

**Comment by PowerfulBacon** [comment]
> Yeah? A shotgun should outmatch a melee weapon? By like definition?

A shotgun will still outmatch a melee weapon, however positioning is going to be an important factor. Walking around like the terminator ignoring everything else happening just clicking on antag to make them horizontal just won't be possible, staying at an effective range where you can land all shots while not being 1 tile away is the new strategy for shotguns.

> They're **literally** designed as close-quarters firearms, they're meant to outmatch a melee weapon. Only a few antagonists rely on melee weapons anyway, that being Slaughter Demons (who can just crawl away from the shotgun), Cult (who has the mirror shield and therefore can only be hurt by slugs, which take good skill to aim with and deal nowhere near as much damage) and e-sword traitors, who could just be sneaky or by a revolver, a literal backpack shotgun (60 brute, same as shotgun slugs).

The mirror shield can only protect against 1 volley, after that it breaks. Slugs do not take a good deal to aim when you are in close range that clicking on a quarter of the tiles visible to you will land a hit, meaning a blind person clicking randomly has a 25% chance of hitting.

 >  which take good skill to aim with and deal nowhere near as much damage

Good skill.
Stole a shotgun and some slugs on a rev round the other day, and just walking into a room on fire unga-bunga style, walking right next to the CE and firing 2 shots killing them before they could react ignoring everything else in the room wasn't exactly what I would have considered to be me doing some high skill gameplay.
And unlike me there are people who are actually robust, and slugs are generally considered to be pretty overpowered. (Nerfing them is a bit more of a challenge without completely negating the way they re meant to be used.)

> Ninjas have ninja-stars, lings can regenerate. Shotguns are meant to outmatch melee weapons, and even then antags who use melee weapons have quite a few counters to them with enough reaction time.

Shotguns are meant to be powerful in close range, rather than outmatch everything with very limited counters.
Lings regenerating is not a counter to shotguns. Their regeneration is slow and usually only useful after combat. They can revive but it takes a long time and by that time they are probably gibbed after getting killed almost instantly by the shotgun.
Space ninja are pretty effective at getting around shotguns, however still at close range 1 shot from a shotgun will slow them down enough to be an easy kill.
Space dragons can be killed by 4 shots from a shotgun, and against anyone at that range will probably get killed. They do have the tailsweep ability, but anyone on a scooter can counter this pretty easily. (Its shockingly easy to kill something that should rule 100% in melee range with a gun)

> Shotguns aren't even a one-hit win they're two hits.

Immediately taking 75 damage is basically next to dead, there is no way you can escape once you get low enough due to damage slowdown and other effects. While not a one shot kill, you can fire them fast enough to get a kill relatively quickly.

> Shotguns are also adaptable as fuck, with different rounds for different antags; slugs for nukies and culties, buckshot for revs / tots, incendiary for lings/blob and sleepy for basically everything else.

Yes indeed, shotguns can be used for any situation. It's almost like something that is incredibly adaptable and can counter everything in the game needs a nerf.
Forgot about slugs in this nerf, they are one of the worst shells and take the near-instant win of the shotgun to all ranges. (Admittedly I never used slugs until recently where I killed the CE and an engineer in a burning room in about 1 second just by shooting, pumping then quickly shooting again (Shotguns are pretty fast-firing if you can use the hotkeys))
Nukies are by design meant to be a powerful threat with powerful armour and are a 3-5 vs 50-70. 1 person with a slug based shotgun shouldn't best the entire nuke team with ease.

> Stop trying to nerf shotgun for doing its job.

Just because something is intentional, doesn't mean it's balanced or fun.
 - Lubes job was to slip people and cause mass annoyance and was nerfed because slipping into a wall and then taking a ton of damage over and over wasn't considered fun and was too annoying.
 - Flux Wave anomlies were intended to destroy large parts of the station if not eliminated and were nerfed because large parts of the station being destroyed was not fun.
 - Firelocks were intended to lock people out but were nerfed because it wasn't fun to be trapped in a room with no pressure needing a crowbar to enter.
 - The cluwne rune's job was to spawn in a cluwne and cause a lot of damage but was removed for doing its job.
 - Toe stubbing's job was to be a nuisance if you weren't wearing shoes but was nerfed for doing its job
 - Organs decay's job was to prevent people from being defibbed after 15 minutes but was nerfed for doing its job
 - Wallslams were nerfed for doing their job
 - Cakehat was nerfed for doing its job
 - Cult instant stun was nerfed for doing its job
 - Holoparasites were nerfed for doing their job
 - macrophage was nerfed for doing its job
 - flashbang was nerfed for doing its job
 - spirit board was nerfed for doing its job
 - lings were nerfed for doing their job
 - wizards have been nerfed multiple times for doing their job
 - contractors were nerfed for doing their job

Something doing its job does not =/= balance, and shotguns are just plain overpowered at this point and should be nerfed the same way all over antags have been.
Play some antags as well as security, being put to near crit immediately by a shotgun is probably the same amount of fun as being touched by a cultist and immediately stunned in an uncounterable way.
It's not like I have something against security specifically.

**Comment by PowerfulBacon** [comment]
> 
> 
> > Yeah? A shotgun should outmatch a melee weapon? By like definition?
> 
> A shotgun will still outmatch a melee weapon, however positioning is going to be an important factor. Walking around like the terminator ignoring everything else happening just clicking on antag to make them horizontal just won't be possible, staying at an effective range where you can land all shots while not being 1 tile away is the new strategy for shotguns.
> 
> > They're **literally** designed as close-quarters firearms, they're meant to outmatch a melee weapon. Only a few antagonists rely on melee weapons anyway, that being Slaughter Demons (who can just crawl away from the shotgun), Cult (who has the mirror shield and therefore can only be hurt by slugs, which take good skill to aim with and deal nowhere near as much damage) and e-sword traitors, who could just be sneaky or by a revolver, a literal backpack shotgun (60 brute, same as shotgun slugs).
> 
> The mirror shield can only protect against 1 volley, after that it breaks. Slugs do not take a good deal to aim when you are in close range that clicking on a quarter of the tiles visible to you will land a hit, meaning a blind person clicking randomly has a 25% chance of hitting.
> 
> > which take good skill to aim with and deal nowhere near as much damage
> 
> Good skill.
> Stole a shotgun and some slugs on a rev round the other day, and just walking into a room on fire unga-bunga style, walking right next to the CE and firing 2 shots killing them before they could react ignoring everything else in the room wasn't exactly what I would have considered to be me doing some high skill gameplay.
> And unlike me there are people who are actually robust, and slugs are generally considered to be pretty overpowered. (Nerfing them is a bit more of a challenge without completely negating the way they re meant to be used.)
> 
> > Ninjas have ninja-stars, lings can regenerate. Shotguns are meant to outmatch melee weapons, and even then antags who use melee weapons have quite a few counters to them with enough reaction time.
> 
> Shotguns are meant to be powerful in close range, rather than outmatch everything with very limited counters.
> Lings regenerating is not a counter to shotguns. Their regeneration is slow and usually only useful after combat. They can revive but it takes a long time and by that time they are probably gibbed after getting killed almost instantly by the shotgun.
> Space ninja are pretty effective at getting around shotguns, however still at close range 1 shot from a shotgun will slow them down enough to be an easy kill.
> Space dragons can be killed by 4 shots from a shotgun, and against anyone at that range will probably get killed. They do have the tailsweep ability, but anyone on a scooter can counter this pretty easily. (Its shockingly easy to kill something that should rule 100% in melee range with a gun)
> 
> > Shotguns aren't even a one-hit win they're two hits.
> 
> Immediately taking 75 damage is basically next to dead, there is no way you can escape once you get low enough due to damage slowdown and other effects. While not a one shot kill, you can fire them fast enough to get a kill relatively quickly.
> 
> > Shotguns are also adaptable as fuck, with different rounds for different antags; slugs for nukies and culties, buckshot for revs / tots, incendiary for lings/blob and sleepy for basically everything else.
> 
> Yes indeed, shotguns can be used for any situation. It's almost like something that is incredibly adaptable and can counter everything in the game needs a nerf.
> Forgot about slugs in this nerf, they are one of the worst shells and take the near-instant win of the shotgun to all ranges. (Admittedly I never used slugs until recently where I killed the CE and an engineer in a burning room in about 1 second just by shooting, pumping then quickly shooting again (Shotguns are pretty fast-firing if you can use the hotkeys))
> Nukies are by design meant to be a powerful threat with powerful armour and are a 3-5 vs 50-70. 1 person with a slug based shotgun shouldn't best the entire nuke team with ease.
> 
> > Stop trying to nerf shotgun for doing its job.
> 
> Just because something is intentional, doesn't mean it's balanced or fun.
> 
>     * Lubes job was to slip people and cause mass annoyance and was nerfed because slipping into a wall and then taking a ton of damage over and over wasn't considered fun and was too annoying.
> 
>     * Flux Wave anomlies were intended to destroy large parts of the station if not eliminated and were nerfed because large parts of the station being destroyed was not fun.
> 
>     * Firelocks were intended to lock people out but were nerfed because it wasn't fun to be trapped in a room with no pressure needing a crowbar to enter.
> 
>     * The cluwne rune's job was to spawn in a cluwne and cause a lot of damage but was removed for doing its job.
> 
>     * Toe stubbing's job was to be a nuisance if you weren't wearing shoes but was nerfed for doing its job
> 
>     * Organs decay's job was to prevent people from being defibbed after 15 minutes but was nerfed for doing its job
> 
>     * Wallslams were nerfed for doing their job
> 
>     * Cakehat was nerfed for doing its job
> 
>     * Cult instant stun was nerfed for doing its job
> 
>     * Holoparasites were nerfed for doing their job
> 
>     * macrophage was nerfed for doing its job
> 
>     * flashbang was nerfed for doing its job
> 
>     * spirit board was nerfed for doing its job
> 
>     * lings were nerfed for doing their job
> 
>     * wizards have been nerfed multiple times for doing their job
> 
>     * contractors were nerfed for doing their job
> 
> 
> Something doing its job does not =/= balance, and shotguns are just plain overpowered at this point and should be nerfed the same way all over antags have been.
> Play some antags as well as security, being put to near crit immediately by a shotgun is probably the same amount of fun as being touched by a cultist and immediately stunned in an uncounterable way.
> It's not like I have something against security specifically.

**Comment by BrotherHangyul** [comment]
I'm literally a security main ._.

"Yes indeed, shotguns can be used for any situation. It's almost like something that is incredibly adaptable and can counter everything in the game needs a nerf."

No? Something being adaptable doesn't mean it should be nerfed, it means it's a good weapon. Anybody with an IQ exceeding bread is able to keep their distance from buckshot and hide from slugs. Slugs deal 60 damage IIRC but they're laughably easy to dodge or otherwise take cover from.

Before you accuse me of never playing antagonists, I have, and shotguns have never been a problem for me. Nukies? Out-gun the fucks. Cult? Kite and dodge the shots. Incursion? Pair up with a friend (e-sword/e-bow.)

And the changes you referenced whilst, are good examples, were due to either a hugbox mindset or just plain overpowered shit rather than "oh no I decided to go fight in melee against a guy with a shotgun and lost"

Mind you, these changes won't do shit either way. A shotgun will still dominate in CQC because (from what I can estimate) 45 brute per buckshot shell is nothing to laugh at. The strongest melee antag weapons (E-sword and cult sword) deal 30-35 damage. Being one tile away from them will still be a valid strategy; the only way to avoid that is to completely nerf shotguns into the ground to the point they're literally nothing.

Stop trying to make security impossible to play, literally every coder here tries to nerf security into the ground every month and it's getting really fucking boring.

Hell, if you're gonna try force this, at least give us a pistol or something (glock from NSV, USP from skyrat etc) so we don't get killed by some DE-sword wielding maniac.

Tl;Dr: Shotguns good, stop nerfing them it's not funny or fun for security players.

**Comment by BrotherHangyul** [comment]
By the way literally every single SS13 server on BYOND hasn't changed the shotgun for good reason.

**Comment by BettonnCZ** [comment]
please, dont. They will nerf sec to the ground :(

**Comment by BrotherHangyul** [comment]
One last thing, nerfing slugs and spread I'm 100% okay with, this is mainly about keeping buckshot buckshott-y.

**Comment by VictorVow** [comment]
Agree with all the changes except to rubbershots, buckshot and two-handedness. I made the sprites huge, anyone can see this gun from a mile away now so if you're still walking up to a shotgun wielder to get hit by all 6 pellets then you deserve that fate. Additionally nerfing rubbershot just encourages using lethal ammo (since at least that damage is permanent) instead, and if you're going to make spread that tiny then the total damage possible should be the same as slugs. 

On a side note I have to agree all the Security nerfs lately are ultra cringe. Flashbangs are virtually useless now for actually dealing with riots and now someone else is also trying to nerf energy guns, so basically all guns will be shit. This is all after **I personally** nerfed stunbatons, so it shows that even this flow of nerfs is never going to end even if the secmains themselves agree to some compromises. 

> security should be going to cargo to arm up and get a good arsenal.

1) Cargo is often less than helpful when you try to order guns for the Security department only. On even greenshifts consistently I've had to deal with shitter Cargonians stealing guns or ordering their own and breaking open the crates.
2) The situation almost always requires an immediate response e.g. cult is snowballing, a ling is EMPing and murdering a seccie, nukies have been spotted, etc
3) HOS is often too incompetent to order stuff in a timely manner

It's not balanced to depend on Cargo for a decent Security response. Every other department can make their own top-level gear, yet weapons at the seclathe are all tradeoffs rather than being direct upgrades of existing weaponry. "Advanced" energy gun for instance can't be recharged in seconds by using an upgraded charger, so it's a tradeoff of having to space out your shots vs needing to visit a charger regularly. 

> Space dragons can be killed by 4 shots from a shotgun, and against anyone at that range will probably get killed. They do have the tailsweep ability, but anyone on a scooter can counter this pretty easily

Are you trying to suggest this is an issue with shotguns rather than it being a scooter exploit? Tailsweeps should knock you off whatever you're buckled to. 

> Play some antags as well as security, being put to near crit immediately by a shotgun is probably the same amount of fun as being touched by a cultist and immediately stunned in an uncounterable way.

I play antags often and like to think I'm semi-decent at it. Whenever I've had to face shotguns it's because I'm already being doing a pretty decent job at antagonising (abducting or killing several seccies usually) for them warrant opening the Armoury. They do pack a punch I won't deny that but also I rarely wear armour when antag. I'd like to see more armour options available (maybe a chameleon armour vest, or an invisible uniform accessory) that can negate shotguns (as well as disabler shots) for antags who attempt to fight Security head-on. 

> 1 person with a slug based shotgun shouldn't best the entire nuke team with ease.

Zesko buffed ballistics fire rate to a ridiculous degree compared to every other server, all nukie weapons can easily down a shotgun user now so I don't get this statement.

**Comment by PowerfulBacon** [comment]
> I made the sprites huge, anyone can see this gun from a mile away now so if you're still walking up to a shotgun wielder to get hit by all 6 pellets then you deserve that fate.

Sprite size isn't a downside when you can instantly pull it out of a bag and fire without them seeing it.

> Additionally nerfing rubbershot just encourages using lethal ammo (since at least that damage is permanent) instead, and if you're going to make spread that tiny then the total damage possible should be the same as slugs.

Shotguns should be a lethal weapon, if you want non lethals use a disabler.
The rubbershot is nerfed on about the same level as the other shells, so they are still stronger damage wise than lethal shells like the buckshot (chart provided in PR desc)
Actually in comparison to buckshot, the non lethal options are better now. (Before the buckshot did significantly more damage than the beanbags and rubbershots, now they do about the same).

> It's not balanced to depend on Cargo for a decent Security response

Good, if they could go to cargo and get a ton of ballistics for any situation like they can go to the armoury to get a shotgun for any situation it would be extremely unbalanced. The use of cargo should be for extreme situations when you need it: Cult, Revs, Nuke ops etc.

> Every other department can make their own top-level gear, yet weapons at the seclathe are all tradeoffs rather than being direct upgrades of existing weaponry.

Like how security can make top-level gear at their lathe (Guns that can shoot through walls, auto-recharging guns, firing pins that can only be used by sec, tesla cannon among others)

> Zesko buffed ballistics fire rate to a ridiculous degree compared to every other server, all nukie weapons can easily down a shotgun user now so I don't get this statement.

Yea true, since of course nuclear operatives need to be about 10-20 times stronger than the average crew member but also that point is to do with slugs which I am still yet to nerf because I am lazy.

**Comment by PowerfulBacon** [comment]
also the two-handedness is the bit I love the most (how does it make sense you can 1 handed pump a shotgun anyway 😄 )

**Comment by BrotherHangyul** [comment]
> also the two-handedness is the bit I love the most (how does it make sense you can 1 handed pump a shotgun anyway 😄 )

Simple.

Flip the shotgun in a 90 degree angle upwards, so that the barrel is facing upwards. Move your hand to the pump. Pump the shotgun. Grab the grip again.

**Comment by Rukofamicom** [comment]
I was worried that this was just going to be a straight nerf to damage when I first saw it. With the tighter spread I like this change a lot and it's exactly what I felt shotguns needed. Current (spread) shotguns feel like they're only actually useable from directly adjacent tiles point-blank, or else you lose almost all of the damage, and this will make slugs more accurate as well.

10/10

**Comment by PowerfulBacon** [comment]
Rebalances without testing are usually pretty shit, going to go put it on MRP and play with shotguns for a bit to see how they roll

**Comment by PowerfulBacon** [comment]
Alright I am very happy with these numbers. Ready for review

File: code/modules/projectiles/projectile/bullets/shotgun.dm [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /obj/item/projectile/bullet/shotgun_slug
 	name = "12g shotgun slug"
 	damage = 60
+	armour_penetration = -60
```
Comment (park66665) [inline code/modules/projectiles/projectile/bullets/shotgun.dm:4]: I... I wasn't told about this...

File: code/modules/projectiles/ammunition/ballistic/shotgun.dm [hunk 2]
```diff
@@ -85,7 +85,7 @@
 	icon_state = "bountyshell"
 	projectile_type = /obj/item/projectile/bullet/pellet/shotgun_incapacitate
 	pellets = 12//double the pellets, but half the stun power of each, which makes this best for just dumping right in someone's face.
-	variance = 25
+	variance = 20
```
Comment (park66665) [inline code/modules/projectiles/ammunition/ballistic/shotgun.dm:88]: > Incapacitating Shell: Reduced stamina damage from 6 to 5. Reduced variance from 25 to 50

**Review by park66665 — COMMENTED** [review]
~~here, have my two cents~~

As always(?), I won't make balance arguments here. I indeed had mained sec for a short while, but I am not going to delve into balance arguments. I indeed heard that shotguns are *almost always* the most desirable weapon, so I guess some balance decision is to be made... like right now.

Anyways, I also built a formula for this and my formula fits quite well to Bacon's so far:

Screenshot 2021-01-18 005710
_Fig. 1 Damage graph for improvised shell with pre-PR stats; this is a different model: x-axis represents literally the distance (in tiles) between the shooter and the target; 1 means the target is right next to the shooter, et cetera_

But you should also keep in mind that the damage you'll be seeing will be c. 15% higher in average, because of this:

Screenshot 2021-01-17 050006
_Fig 2. The angle sensitivity of expected shotgun damage_

In short, Bacon's and my formula assumes theta 2, but you can clearly see that the hit angle actually is wider at theta 1 *while having the same distance*. That means, you get a somewhat higher damage than the bacon's numbers if the target is diagonally away, for greater the hitting angle, greater the expected damage. I actually haven't calculated it, but I am going to guessimate that the average additional damage here is around 15%. So, when you see the Bacon's spreadsheet, you can just add 15% to see the actual damage. Hope that clears the confusion and lets you see the real situation.

File: code/modules/projectiles/projectile/bullets/shotgun.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/modules/projectiles/projectile/bullets/shotgun.dm:4]: ah yea let me update desc

File: code/modules/projectiles/ammunition/ballistic/shotgun.dm (hunk 2 above)
Comment (PowerfulBacon) [reply code/modules/projectiles/ammunition/ballistic/shotgun.dm:88]: oh should say 20 oops

**Comment by PowerfulBacon** [comment]
From the testmerge and a subsequent rev round involving running round with a pipe shotgun
 - HOP went down in about 3 buckshot shells (Pretty standard armour)
 - Captain went down in 4 buskshot shells (Pretty good armour)
 - HOS was set on fire by incendiary shells but I was too in the action to count the shots
 - Shotguns were much better at low-mid ranges rather than point blank due to the tighter spread.

Overall I'd say a very good improvement over what we had and fulfilled the purpose of changing their role in combat without nullifying their purpose too significantly.

**Review by utilbird — APPROVED** [review]

**Comment by LivingBatteryBox** [comment]
Honestly only part of this I don't like is the slugs having less armor penetration bit.

**Pull request merged**